dev/mock_source_data.py

```diff
@@ -0,0 +1,165 @@
+from os import environ, path
+
+import boto3
+from flask import Flask, request
+from flask_api import status
+
+
+FLASK_HOST = environ.get("FLASK_HOST", "0.0.0.0")
+FLASK_PORT = environ.get("FLASK_PORT", 5000)
+
+LOCALSTACK_URL = environ.get("AWS_ENDPOINT", "http://localstack:4566")
+BUCKET_NAME = environ.get("RETRIEVAL_BUCKET_NAME", "epid-sources-raw")
+S3_BUCKET_URL = f"s3://{BUCKET_NAME}"
+
+SOURCE_DETAILS = {
+	"origin": {
+		"url": ""
+	},
+	"format": "",
+	"automation": {
+		"parser": {
+			"awsLambdaArn": "",
+		},
+	},
+	"dateFilter": "",
+	"hasStableIdentifiers": False
+}
+
+SOURCE_NAMES_TO_PARSERS = {
```

Comment:
OK for now, but ideally this should be taken automatically from the parsing/ directory.

Reply:
IMHO we should refactor some of our Python parser/ingestor code and make it into a library, then we can simply import that library and call a function to give us that.
Relying on relative pathing and breaking dependency inversion is bad, and so is hardcoding this dict.
In the setup_ingestion service I mounted "../ingestion/functions:/ingestion/functions" as a volume...so importing a function from a library and calling it will shrink LOC and our Docker images, and make them less brittle (and probably yield other benefits).